index.html

```diff
@@ -112,7 +112,285 @@
+          "@type": "Person",
+          "name": "Erik Bjare",
+          "url": "https://erik.bjareholt.com/",
+          "sameAs": "https://www.linkedin.com/in/erikbjareholt/"
+      },
+      "maintainer": [
+          {
+              "@type": "Person",
+              "name": "Erik Bjare",
+              "url": "https://erik.bjareholt.com/",
+              "sameAs": "https://www.linkedin.com/in/erikbjareholt/"
+          },
+          {
+              "@type": "Person",
+              "name": "Johan Bjäreholt",
+              "url": "https://www.linkedin.com/in/johanbjareholt/"
+          }
+      ],
+      "contributor": [
+          {
+              "@type": "Person",
+              "name": "Brayo",
+              "url": "https://github.com/brayo-pip"
+          },
+          {
+              "@type": "Person",
+              "name": "Erik Marforio",
+              "url": "https://github.com/emarforio"
+          },
+          {
+              "@type": "Person",
+              "name": "Måns Magnusson",
+              "url": "https://github.com/exoji2e"
+          },
+          {
+              "@type": "Person",
+              "name": "Nicolae Stroncea",
+              "url": "https://github.com/nicolae-stroncea"
+          }
+      ],
+      "funding": [
+          {
+              "@type": "Grant",
+              "name": "Finnish Unix User Group (FUUG)",
+              "url": "https://fuug.fi/"
+          },
+          {
+              "@type": "MonetaryGrant",
+              "name": "Ghent University",
+              "url": "https://www.ugent.be/en"
+          }
+      ],
+      "sponsor": [
+          {
+              "@type": "Person",
+              "name": "Example Sponsor",
+              "url": "https://github.com/dgudim"
+          },
+          {
+              "@type": "Person",
+              "name": "Alge97",
+              "url": "https://github.com/Alge97"
+          },
+          {
+              "@type": "Person",
+              "name": "Dmitri Vassilenko",
+              "url": "https://github.com/faevourite"
+          },
+          {
+              "@type": "Person",
+              "name": "Eugene Dobry",
+              "url": "https://github.com/edobry"
+          },
+          {
+              "@type": "Person",
+              "name": "Justyn Butler",
+              "url": "https://github.com/justyn"
+          },
+          {
+              "@type": "Person",
+              "name": "Jared Wolff",
+              "url": "https://github.com/jaredwolff"
+          },
+          {
+              "@type": "Organization",
+              "name": "cynkra",
+              "url": "https://github.com/cynkra"
+          },
+          {
+              "@type": "Person",
+              "name": "Tymoteusz Czech",
+              "url": "https://github.com/Tymek"
+          },
+          {
+              "@type": "Person",
+              "name": "Glenn Willen",
+              "url": "https://github.com/gwillen"
+          }
+      ],
```

Comment:
These seem kinda random. If they are to be kept, they should prob reflect https://activitywatch.net/sponsors/